test/inductor/test_templated_attention.py

```diff
@@ -231,7 +236,7 @@ def sdpa_hop(q, k, v, score_mod):
         # x_ref      = ∑_i e^(scores[i])
         # x_compiled = ∑_i 2^(log2(e) * scores[i])
 
-        self.assertTrue(ref_lse.dtype == torch.float32)
+        self.assertTrue(ref_lse.dtype == torch.float64)
```

Comment:
We updated the working-precision to the fp32 if the inputs are <= fp32 | or fp64 if the inputs are fp64 which is true for the ref in this case